videointelligence/video_analyze/video_analyze_test.go

```diff
@@ -47,9 +48,9 @@ func TestAnalyze(t *testing.T) {
 				t.Fatal("gcs not set")
 			}
 
+			time.Sleep(5 * time.Second) // Give buffer between tests for quota.
```

Comment:
Just out of curiosity-- what is the quota? Why does it seem better to add sleep after every test rather than retry in the case of error?

Reply:
Retry definitely seems better! Switched.

Reply:
I also updated all of the videointelligence tests, not just this one, since they all seem to run into the out of quota error.